test/basic_test.rb

```diff
@@ -196,11 +196,11 @@ def test_multi_polygon_column
     end
     SpatialModel.reset_column_information
     rec = SpatialModel.new
-    wkt = "MULTIPOLYGON (((-73.97210545302842 40.782991711401195, " \
+    wkt = "MULTIPOLYGON (((-73.97210545302842 40.78299171140119, " \
```

Comment:
Why this change ?

Reply:
I'm not sure. For whatever reason the 15th decimal in this wkt was getting truncated on the output. I tried looking into it a bit, but I'm not sure if it's a PostGIS change or something to do with the WKT parsers in RGeo. I'll see if I can figure out exactly why it's failing.

Also kind of related but this PR (https://github.com/rgeo/activerecord-postgis-adapter/pull/373) pointed out that we use the Ruby based parsers here. I can open a separate PR that uses the Geos based parsers if available.

Reply:
@BuonOmo it works now so i'm not sure what was going on before.